### DISCUSSION TITLE
Adding WTF support to blacksheep

### DISCUSSION THREAD

**Discussion by Legopapurida** [Neoteroi/BlackSheep#143] [Ideas]
Hello to blacksheep community.
I am a flask developer and I am in doubt with choosing blacksheep over the fastapi coz blacksheep is newcomer to python web development and there is not many extensions like flask or django for blacksheep.
I will be grateful that you add WTF supporting for blacksheep.

I loved your project more than fastapi and I benchmarked all python web servers with apache bench, at end I realized that blackshepp is really cool but if you add new features this will replace fastapi as soon as

**Comment by myusko** [comment]
Hi, @Legopapurida 

Thank you for such kind words from your side, and also thank you for your suggestions.

This totally makes sense to have more extensions, and they will be for sure in the future, but right now we're putting our efforts into new features such as #117 

Right now we have the [extension for SQLAlchemy](https://github.com/Neoteroi/BlackSheep-SQLAlchemy)

Thanks,
Mykhailo
